whipper/common/program.py

```diff
@@ -447,6 +454,9 @@ def getTagList(self, number, mbdiscid):
                 tags['MUSICBRAINZ_ALBUMID'] = mbidRelease
                 tags['MUSICBRAINZ_RELEASEGROUPID'] = mbidReleaseGroup
                 tags['MUSICBRAINZ_ALBUMARTISTID'] = mbidReleaseArtist
+                if classical and isClassical:
+                    tags['COMPOSER'] = trackArtist
+                    tags['PERFORMER'] = recordingArtist
```

Comment:
Eh. I’m not a big fan of this. I feel like we should get this information from relationships if we want to get this. This could maybe be useful as a fallback, but the relationships seem like it should be the first source.

Reply:
I have changed the way to get composer(s) and performer(s) in my complete new commits. Thanks !